lib/CMakeLists.txt

```diff
@@ -51,12 +51,13 @@ endif()
 
 # The rest of the build
 
-include_directories(${CMAKE_CURRENT_BINARY_DIR}/../include)
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/..)
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../include)
-include_directories(${CMAKE_CURRENT_BINARY_DIR}/..)
-include_directories(${CMAKE_CURRENT_SOURCE_DIR})
-include_directories(${CMAKE_CURRENT_BINARY_DIR})
+include_directories(
+  ${CURL_BINARY_DIR}/include/curl # for generated curlbuild.h from curl.h
```

Comment:
using `CMAKE_CURRENT_XXX_DIR` as opposed to `CMAKE_XXX_DIR` is the recommended way to support standalone and part of larger project builds. The current version will then point to e.g curls binary dir and not the one for the whole project.

Maybe it's better to be consistent and use the `CURL_BINARY_DIR` variable until/if all usages are updated.